distrobox-assemble

```diff
@@ -437,9 +437,11 @@ run_distrobox()
 	fi
 	if [ -n "${volume}" ]; then
 		IFS="¤"
+		args=""
 		for vol in ${volume}; do
```

Comment:
We can just implement this: https://github.com/89luca89/distrobox/issues/1514#issuecomment-2279202367